prometheus-operator.yaml

```diff
@@ -24,9 +24,12 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 52526e3b0cfb5f3deaa967998ff88a5133f1ec0f
 
-  - uses: autoconf/make
-    with:
-      opts: operator
+  - runs: |
+      # Handle CVE-2023-39325 and CVE-2023-3978
+      go get golang.org/x/net@v0.17.0
+      go mod tidy
+
+      make operator
```

Comment:
We could keep the `autoconf/make` pipeline and just add a new step before

Reply:
thanks for the pointer, done